components/icons/images/tier1/volume.svg

```diff
@@ -1,8 +1,3 @@
-<svg width="18" height="18" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 18 18" mirror-in-rtl="true">
-  <mask id="a" fill="#fff">
-    <path d="M7.003 3.835v10.421c0 .189-.112.36-.285.436a.424.424 0 01-.19.038.526.526 0 01-.321-.123l-2.915-2.719H.932a.93.93 0 01-.928-.928V7.132a.93.93 0 01.928-.928h2.35l2.925-2.72a.483.483 0 01.511-.084.478.478 0 01.285.435zM13.44.828c.07.07 6.876 7.695.047 16.38a.947.947 0 11-1.495-1.167 10.73 10.73 0 00.046-13.936A.95.95 0 0113.44.828zm-2.865 2.818a8.053 8.053 0 012.162 5.4 8.053 8.053 0 01-2.16 5.4.953.953 0 01-1.338.019.947.947 0 01-.028-1.336 6.407 6.407 0 001.62-4.083 6.407 6.407 0 00-1.62-4.083.947.947 0 011.364-1.317z"/>
-  </mask>
-  <g mask="url(#a)">
-    <path fill="#494c4e" d="M-1 0h18v18H-1z"/>
-  </g>
+<svg width="18" height="18" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 18 18">
+  <path d="M7.003 3.835v10.421c0 .189-.112.36-.285.436a.424.424 0 0 1-.19.038.526.526 0 0 1-.321-.123l-2.915-2.719H.932a.93.93 0 0 1-.928-.928V7.132a.93.93 0 0 1 .928-.928h2.35l2.925-2.72a.483.483 0 0 1 .511-.084.478.478 0 0 1 .285.435zM13.44.828c.07.07 6.876 7.695.047 16.38a.947.947 0 1 1-1.495-1.167 10.73 10.73 0 0 0 .046-13.936A.95.95 0 0 1 13.44.828zm-2.865 2.818a8.053 8.053 0 0 1 2.162 5.4 8.053 8.053 0 0 1-2.16 5.4.953.953 0 0 1-1.338.019.947.947 0 0 1-.028-1.336 6.407 6.407 0 0 0 1.62-4.083 6.407 6.407 0 0 0-1.62-4.083.947.947 0 0 1 1.364-1.317z"/>
```

Comment:
```suggestion
  <path fill="#494c4e" d="M7.003 3.835v10.421c0 .189-.112.36-.285.436a.424.424 0 0 1-.19.038.526.526 0 0 1-.321-.123l-2.915-2.719H.932a.93.93 0 0 1-.928-.928V7.132a.93.93 0 0 1 .928-.928h2.35l2.925-2.72a.483.483 0 0 1 .511-.084.478.478 0 0 1 .285.435zM13.44.828c.07.07 6.876 7.695.047 16.38a.947.947 0 1 1-1.495-1.167 10.73 10.73 0 0 0 .046-13.936A.95.95 0 0 1 13.44.828zm-2.865 2.818a8.053 8.053 0 0 1 2.162 5.4 8.053 8.053 0 0 1-2.16 5.4.953.953 0 0 1-1.338.019.947.947 0 0 1-.028-1.336 6.407 6.407 0 0 0 1.62-4.083 6.407 6.407 0 0 0-1.62-4.083.947.947 0 0 1 1.364-1.317z"/>
```

Reply:
Nice catch, thanks .. not sure how those got stripped